src/Terminal/commands/alias.ts

```diff
@@ -6,6 +6,10 @@ export function alias(args: (string | number | boolean)[]): void {
     printAliases();
     return;
   }
+  if (args[0] === "--all") {
+    Terminal.print(`--all is reserved for removal`);
```

Comment:
This should be displayed w/ Terminal.error, since alias is failing to do what the player asked it to do and the message is telling the player why it failed.